src/sentry/models/groupregression.py

```diff
@@ -0,0 +1,23 @@
+from django.db import models
+from django.utils import timezone
+
+from sentry.db.models import FlexibleForeignKey, Model, sane_repr
+
+
+class GroupRegression(Model):
+    """
+    Describes when a group was marked as a regression.
+    """
+
+    __include_in_export__ = False
+
+    group = FlexibleForeignKey("sentry.Group")
+    # the release in which this regressed in
+    release = FlexibleForeignKey("sentry.Release")
+    date_added = models.DateTimeField(default=timezone.now, db_index=True)
```

Comment:
what query pattern does`db_index` on date_added here help with? does it need to be grouped index?

Reply:
Hmm...I could be wrong, but I'm thinking most queries could benefit from this as an index. I think for the various graphs over time, we could see performance gains by restricting date_added. Might be beneficial to include date_added in queries for a certain release as well?

Now I'm also thinking an index on release would be really helpful too and make the primary `regressed_in:release` search a lot quicker.

Reply:
Should you have a unique constraint on (group, release) or can a group regress multiple times in a single release?

Reply:
> Hmm...I could be wrong, but I'm thinking most queries could benefit from this as an index
Could you give an example of how we might be querying on this column? My understanding so far is that this will only ever be used to search with `regressed_in:release`, and will then produce a list of groups. In that case, the `date_added` index won't be used at all, since we'll just be using the `release` index.

> Now I'm also thinking an index on release would be really helpful too and make the primary regressed_in:release search a lot quicker.
Since it's a foreign key, it automatically will have an index added